config/environments/production.rb

```diff
@@ -13,6 +13,7 @@
   config.assets.js_compressor = :uglifier
   config.assets.compile = false
   config.assets.digest = true
+  config.assets.gzip = false
```

Comment:
do we need this in `development.rb` or `test.rb`?

Reply:
> do we need this in `development.rb` or `test.rb`?

Now that you mention it, it might be good at least for `test.rb`, since that could help in CircleCI, where it [precompiles](https://github.com/18F/identity-idp/blob/a316c8706320e353fb5bb073440153f62d496142/.circleci/config.yml#L172) in the [`test` env](https://github.com/18F/identity-idp/blob/a316c8706320e353fb5bb073440153f62d496142/.circleci/config.yml#L20). Since it primarily benefits asset precompilation, I don't know that it would have much benefit for development?

I was initially inclined to put it in `config/application.rb` since it seemed to make sense as "common", though it seemed like a better fit here with the other asset configuration.

Reply:
I agree that finding this setting next to the other settings makes a ton of sense, so having it here in `production.rb` seems good enough, even if it is the same across all envs

Reply:
👍  Kept it env-specific, but added it to `test.rb` as well in f3dfb9b, per note about CircleCI in https://github.com/18F/identity-idp/pull/5812#discussion_r783417430.